hivemind_bus_client/intercom_utils.py

```
@@ -0,0 +1,97 @@
+
+    Returns:
+        IntercomVerificationResult with verification details.
+    """
+    sender_pubkey = pload.get("sender_pubkey")
+    if not sender_pubkey:
+        return IntercomVerificationResult(signature_valid=False, trusted=False)
+
+    try:
+        ciphertext = pybase64.b64decode(pload["ciphertext"])
+        signature = pybase64.b64decode(pload["signature"])
+    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)